tests/fast-integration/test/fixtures/loki/loki.yaml

```diff
@@ -0,0 +1,269 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: loki-test
+  name: logging-loki-test
+  namespace: kyma-system
+automountServiceAccountToken: true
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: logging-loki-test
+  namespace: kyma-system
+  labels:
+    app: loki-test
+data:
```

Comment:
Use `stringData` so that we can have the loki.yaml unencoded.

Reply:
config data changed to plain text